### PR TITLE
Update Readme A3 & A4

### LINKED ISSUE #17: Update Readme A3
Update readme to acknowledge the original source of the code

### LINKED ISSUE #18: Update Readme A4
Update all remaining contents of the Readme page to match our own project

### PR DESCRIPTION
**Pull request by miuna9890**
Update Readme A3 and A4
Acknowledge source and update other contents
Fixes #17 
Fixes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,12 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+* This is **an address book application targeted for private school teachers and educators**.<br>
```
Comment (MinkhantEain) [inline README.md:5]: I don't think it will be considered as address book anymore. Maybe we can discuss it in meeting.

**Review by MinkhantEain — COMMENTED** [review]
Aside from the comments, everything else looks good :+1:

File: README.md (hunk 1 above)
Comment (zhou-colla) [reply README.md:5]: Added to things to discuss in the collab notes

**Review by zhou-colla — APPROVED** [review]
Looks good, further discussion may be required on the production definition, where it is a address book or not

**Pull request merged**